README.md

```diff
@@ -1,48 +1,63 @@
 
-## Features
+### Stable Version
 
-Completions with snippet support.
+* Open the command palette and run `Package Control: Install Package` then select `LSP`.
 
-Navigate code with `Go to Symbol Definition` and `Find Symbol References`
 
-Inline documentation from Hover and Signature Help popups
+### Development Version
 
-![hover screenshot](https://raw.githubusercontent.com/tomv564/LSP/master/docs/images/screenshot-hover.png)
+* Clone this repository into your Packages directory.
+* Open the command palette and run `Package Control: Satisfy Dependencies`.
 
-As-you-type diagnostics with support for code fixes (`F4` to select, `super+.` to trigger actions)
 
-![diagnostics screenshot](https://raw.githubusercontent.com/tomv564/LSP/master/docs/images/screenshot-diagnostics-action.png)
+## Getting started
 
-## Installing
+Follow the installation steps for a <a href="#available_languages">language server</a>.
 
-Normal installation: Install the **LSP** package through Package Control in Sublime Text.
+Open a document supported by the language server. LSP should report the language server starting in the status bar.
 
-To run latest master:
-1. Clone this repository into your Packages directory
-2. Run `Package Control: Satisfy Dependencies` in Sublime
+Documentation is available at [LSP.readthedocs.io](https://LSP.readthedocs.io).
 
-## Configuration
 
-1. Install a language server for a language of your choice
-2. Run `LSP: Enable Language Server` from the Command Palette to enable your server.
-  * A configuration can be added under `clients` in `Preferences: LSP Settings`
-3. Open a document supported by this language server.
-4. LSP should report the language server starting in the status bar.
+## Available Languages <a name="available_languages"></a>
+* [Bash](https://lsp.readthedocs.io/en/latest/#bash)
+* [C/C++](https://lsp.readthedocs.io/en/latest/#clangd)
+* [CSS/LESS/SASS (SCSS only)](https://lsp.readthedocs.io/en/latest/#css)
+* [Dart](https://lsp.readthedocs.io/en/latest/#dart)
+* [Flow (JavaScript)](https://lsp.readthedocs.io/en/latest/#flow)
+* [Go](https://lsp.readthedocs.io/en/latest/#go)
+* [HTML](https://lsp.readthedocs.io/en/latest/#html)
+* [Java](https://lsp.readthedocs.io/en/latest/#java)
+* [JavaScript/TypeScript](https://lsp.readthedocs.io/en/latest/#typescript)
+* [JSON](https://lsp.readthedocs.io/en/latest/#json)
+* [Julia](https://lsp.readthedocs.io/en/latest/#julia)
+* [Kotlin](https://lsp.readthedocs.io/en/latest/#kotlin)
+* [PHP](https://lsp.readthedocs.io/en/latest/#php)
+* [Polymer](https://lsp.readthedocs.io/en/latest/#polymer)
+* [Python](https://lsp.readthedocs.io/en/latest/#python)
+* [R](https://lsp.readthedocs.io/en/latest/#r)
+* [Reason](https://lsp.readthedocs.io/en/latest/#reason)
+* [Ruby](https://lsp.readthedocs.io/en/latest/#ruby)
+* [Rust](https://lsp.readthedocs.io/en/latest/#rust)
+* [Scala](https://lsp.readthedocs.io/en/latest/#scala)
+* [Vue (JavaScript)](https://lsp.readthedocs.io/en/latest/#vue)
```

Comment:
The fragment parts of these URLs don't work (e.g. https://lsp.readthedocs.io/en/latest/#java does not bring my browser to the "java" setup section. Do they work for your browser?

Reply:
Oh right, thanks for explaining that :)